scripts/e2e-iOS.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+export SAUCE_USERNAME=okta-qa
+export SAUCE_ACCESS_KEY="$(aws s3 --quiet --region us-east-1 cp s3://ci-secret-stash/prod/saucelabs/saucelabs_access_key /dev/stdout)"
+export SAUCE_PLATFORM_NAME="iOS";
+export TRAVIS=true # work-around to run tests on saucelabs instead of chrome
+export TRAVIS_JOB_NUMBER=${TEST_SUITE_ID}
+export TRAVIS_BUILD_NUMBER=${TEST_SUITE_RESULT_ID}
+
+cd ${OKTA_HOME}/${REPO}
+
+# Download and start sauce connect
+curl -o ${OKTA_HOME}/${REPO}/sc-4.5.3-linux.tar.gz https://saucelabs.com/downloads/sc-4.5.3-linux.tar.gz
+tar -xzf ${OKTA_HOME}/${REPO}/sc-4.5.3-linux.tar.gz
+${OKTA_HOME}/${REPO}/sc-4.5.3-linux/bin/sc -u ${SAUCE_USERNAME} -k ${SAUCE_ACCESS_KEY} -i ${TRAVIS_JOB_NUMBER} &
+
+aws s3 --quiet --region us-east-1 cp s3://ci-secret-stash/prod/signinwidget/test_credentials ./test_credentials.yaml
```

Comment:
Just to double check - do we need to pass in a  [`--profile`](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html)?

Reply:
We don't need to. This is the format we've used in other scripts too. Working fine... 🤷‍♂